Pre-processing/PlotAllChannels.py

```diff
@@ -1,21 +1,38 @@
 import numpy as np
 import matplotlib.pyplot as plt
 from mne.io import concatenate_raws, read_raw_edf
+import os
 
 
 #Load Data
-load_path = 'C:/Users/User/Desktop/CSE 481C - Neruo Capstone/Project Repo/Data/TUH/00003010_s003_t000.edf'
+filename = '00003010_s003_t000'
```

Comment:
I guess it's fine for now, but when I integrate with front end I want to pull this filename from the api call instead of hardcoded